src/biokbase/narrative/tests/generate_test_results.py

```diff
@@ -192,18 +192,21 @@ def generate_job_logs(all_jobs):
     """
     job_logs = generate_bad_jobs()
     for job_id in all_jobs:
+        test_job = get_test_job(job_id)
+        batch_job = test_job.get("batch_job", JOB_ATTR_DEFAULTS["batch_job"])
+        batch_id = job_id if batch_job else get_test_job(job_id).get("batch_id", None)
```

Comment:
ensure that the batch parent job has the batch ID set -- unfortunately this slipped through because ee2 doesn't set the batch ID for the batch parent job.